README.md

```diff
@@ -55,6 +55,7 @@ An [Ansible AWX](https://github.com/ansible/awx) operator for Kubernetes built w
             * [Cluster-scope to Namespace-scope considerations](#cluster-scope-to-namespace-scope-considerations)
             * [Project is now based on v1.x of the operator-sdk project](#project-is-now-based-on-v1x-of-the-operator-sdk-project)
             * [Steps to upgrade](#steps-to-upgrade)
+      * [Disable IPV6](#IPV6)
```

Comment:
To make this link work properly, you'll need:


```suggestion
      * [Disable IPV6](#disable-ipv6)
```

Reply:
Thank you very much @rooftopcellist  can you please check now.